readme.md

```diff
@@ -94,6 +94,10 @@ Whether the task must finish in the given interval or will be carried over into
 
 `PQueue` instance.
 
+### concurrency
+
+Get/set the concurrency limit.
+
```

Comment:
If a setter, need to document Type, defaults, etc. 
Method would need to be similarly documented.